### PR TITLE
NCM Dashboard

### PR DESCRIPTION
**Pull request by awisema**
`_patient_card.haml` needs some love before this can be finished.

### CODE REVIEW

File: drivers/claims_reporting/app/models/claims_reporting/calculators/patient_risk_score.rb [hunk 1]
```diff
@@ -0,0 +1,159 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module ClaimsReporting::Calculators
+  class PatientRiskScore
+    def initialize(medicaid_ids = [], date_range: Date.current.beginning_of_year..Date.current.end_of_year)
```
Comment (eanders) [inline drivers/claims_reporting/app/models/claims_reporting/calculators/patient_risk_score.rb]: Does it matter that this goes out into the future?

**Review by eanders — COMMENTED** [review]
Looks good to me.  I especially like the Hl7 class, seems like a big win.

File: drivers/claims_reporting/app/models/claims_reporting/calculators/patient_risk_score.rb (hunk 1 above)
Comment (awisema) [reply drivers/claims_reporting/app/models/claims_reporting/calculators/patient_risk_score.rb]: It was done this way to be compatible with what the report did by default. That said, it does mean that the risk scores are going to be weird the beginning of the year on the NCM dashboard -- I'll look at making this a sliding window or something...

Comment (awisema) [reply drivers/claims_reporting/app/models/claims_reporting/calculators/patient_risk_score.rb]: This turned out to be the wrong risk score, so, leaving this as is, just not using it in the UI.

**Review by eanders — CHANGES_REQUESTED** [review]
@awisema, can you make this work for My Agency's Patients as well? It looks slightly more difficult than just updating the partial.

**Pull request merged**